forge/comms/devices.js

```
@@ -110,6 +109,11 @@ class DeviceCommsHandler {
                 }
             } catch (err) {
                 // Not a JSON payload - ignore
+                if (err instanceof SyntaxError) {
```

Was previously swallowing all exceptions, even those unrelated to JSON parsing.